domains/migrations/0003_auto_20160203_1758.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.2 on 2016-02-03 17:58
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('domains', '0002_auto_20160202_2029'),
```

Comment:
Could squash all these into a single migration for the initial merge/commit of the app.